### PR TITLE
Moving to commands and trying to slim down the server start script

### PR DESCRIPTION
**Pull request by jakeswenson**
Helps with #66

### CODE REVIEW

File: core/src/main/java/io/paradoxical/cassieq/commands/ConfigDumpCommand.java [hunk 1]
```diff
@@ -5,61 +5,76 @@
...
+    private static final String DefaultConfigFile = "/data/conf/configuration.yml";
 
     public ConfigDumpCommand() {
-        super("config", "Prints out default config settings for use in creating custom config files");
+        super("config", "Prints out the config settings for use in creating custom config files");
+    }
+
+    @Override
+    public void run(final Bootstrap<?> wildcardBootstrap, final Namespace namespace) throws Exception {
+        wildcardBootstrap.setConfigurationFactoryFactory(
```
Comment (devshorts) [inline core/src/main/java/io/paradoxical/cassieq/commands/ConfigDumpCommand.java]: What does this do?

File: core/src/main/java/io/paradoxical/cassieq/commands/ConfigDumpCommand.java [hunk 2]
```diff
@@ -5,61 +5,76 @@
...
 
-        yamlFactory.configure(YAMLGenerator.Feature.USE_NATIVE_TYPE_ID, false);
+            return;
+        }
 
-        final ObjectMapper objectMapper = new ObjectMapper(yamlFactory);
+        @Cleanup final FileReader fileReader = new FileReader(configFile == null ? DefaultConfigFile : configFile);
 
-        objectMapper.configure(SerializationFeature.INDENT_OUTPUT, true);
+        new BufferedReader(fileReader).lines().forEach(System.out::println);
```
Comment (devshorts) [inline core/src/main/java/io/paradoxical/cassieq/commands/ConfigDumpCommand.java]: Can we use a `System.out.printlin(FileUtils.readFileToString(file))`  instead?

File: core/src/main/java/io/paradoxical/cassieq/commands/HelpCommand.java [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    }
+
+    @Override
+    public void run(final Bootstrap<?> bootstrap, final Namespace namespace) throws Exception {
+        final String helpTextPath = "/data/content/help.txt";
+
+        @Cleanup final FileReader fileReader = new FileReader(helpTextPath);
+
+        new BufferedReader(fileReader).lines().forEach(System.out::println);
+        }
```
Comment (devshorts) [inline core/src/main/java/io/paradoxical/cassieq/commands/HelpCommand.java]: bracket formatting. same comment re file utils

**Comment by devshorts** [comment]
just want the formatting done, otherwise approved

File: core/src/main/java/io/paradoxical/cassieq/commands/ConfigDumpCommand.java (hunk 2 above)
Comment (jakeswenson) [inline core/src/main/java/io/paradoxical/cassieq/commands/ConfigDumpCommand.java]: used guava files instead

File: core/src/main/java/io/paradoxical/cassieq/commands/HelpCommand.java (hunk 3 above)
Comment (jakeswenson) [inline core/src/main/java/io/paradoxical/cassieq/commands/HelpCommand.java]: fixed

**Pull request merged**